NDTensors/src/blocksparse/blocksparsetensor.jl

```diff
@@ -242,7 +242,8 @@ end
 
 @propagate_inbounds function getindex(T::BlockSparseTensor{ElT,0}) where {ElT}
   nnzblocks(T) == 0 && return zero(ElT)
-  return storage(T)[]
+  return getindex(expose(storage(T)))
```

Comment:
@mtfishman This line was making `Metal` testing fail for NDTensors because calling `getindex(storage)` does scalar indexing. Would this line be okay to include in this PR?

Reply:
Sure, though I would write it as `expose(storage(T))[]`.